### PR TITLE
Describe arbitrary attributes in the README

### PR DESCRIPTION
**Pull request by dasch**
@medcat can you take a look at this? Does it adequately describe the feature?

### CODE REVIEW

**Comment by dasch** [comment]
@bquorning can you give some feedback here?

File: README.md [hunk 1]
```diff
@@ -152,7 +152,27 @@ All argument values will be strings. A compilation error will be raised if
...
+
+```html
+{{greetings alice=hello bob=hi}}
+<!-- The above would be rendered as: -->
+alice: hello!
+bob: hi!
+```
+
+Note that since keyword arguments in Ruby are represented as Symbol objects, which are
+not garbage collected, accepting arbitrary attributes represents a security
```
Comment (bquorning) [inline README.md]: > “as Symbol objects, which are not garbage collected”

They are, in Ruby 2.2.

Comment (dasch) [inline README.md]: I'll qualify the statement, then.

**Comment by bquorning** [comment]
Looks good :+1:

**Comment by medcat** [comment]
I would note that if you define a required keyword, it is still required even if other keyword arguments are allowed.

**Pull request merged**